### DISCUSSION TITLE
Improve filter component to enable an Apply button.

### DISCUSSION THREAD

**Discussion by chapeaupaul** [evidence-dev/evidence#2880] [Ideas]
### Description

Why?
Currently, when using multiple selections for a filter, if the user wants to select a single value, they need to unselect all first, which automatically updates the dashboard. This results in missing data across the dashboard and creates a poor user experience for end users. 

It could be done at the filter level or as one button for all available filters in the reports.

**Comment by rudnik1991** [comment]
+1 Really need this functionality in the filter.

**Comment by archiewood** [comment]
Is this just for the Dropdown component, or many components?
